Firefox (75) and AOM itself fail to build with NEON enabled. As such
we should disable it for now.

In file included from /home/buildozer/aports/community/firefox/src/firefox-75.0/third_party/aom/aom_dsp/arm/blend_a64_mask_neon.c:12:
/home/buildozer/aports/community/firefox/src/firefox-75.0/third_party/aom/av1/common/arm/mem_neon.h: In function 'load_u8_8x8':
/usr/lib/gcc/armv7-alpine-linux-musleabihf/9.3.0/include/arm_neon.h:10303:1: error: inlining failed in call to always_inline 'vld1_u8': target specific option mismatch
10303 | vld1_u8 (const uint8_t * __a)
      | ^~~~~~~
--- a/media/libaom/config/linux/arm/config/aom_config.asm.orig	2020-04-08 21:58:08.152208663 +0200
+++ b/media/libaom/config/linux/arm/config/aom_config.asm	2020-04-08 21:58:30.775311580 +0200
@@ -65,7 +65,7 @@
 .equ HAVE_MIPS64, 0
 .equ HAVE_MMX, 0
 .equ HAVE_MSA, 0
-.equ HAVE_NEON, 1
+.equ HAVE_NEON, 0
 .equ HAVE_SSE, 0
 .equ HAVE_SSE2, 0
 .equ HAVE_SSE3, 0
--- a/media/libaom/config/linux/arm/config/aom_config.h.orig	2020-04-08 21:58:16.878786419 +0200
+++ a/media/libaom/config/linux/arm/config/aom_config.h	2020-04-08 21:58:23.078723273 +0200
@@ -67,7 +67,7 @@
 #define HAVE_MIPS64 0
 #define HAVE_MMX 0
 #define HAVE_MSA 0
-#define HAVE_NEON 1
+#define HAVE_NEON 0
 #define HAVE_SSE 0
 #define HAVE_SSE2 0
 #define HAVE_SSE3 0
